wrappers/services_gen_test.go

```diff
@@ -646,9 +648,6 @@ Restart=%s
 WorkingDirectory=/var/snap/snap/44
 TimeoutStopSec=30
 Type=%s
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
Looks like we had a bug where the service triggered by a timer would get started during boot. On the upside, it would only actually run if the attempt was inside the schedule window.

Reply:
We never actually ran `systemctl enable` on these units.  The purpose of this change is (1) stop the units from being reported as "disabled", and (2) so the user doesn't incorrectly enable the units.